### PR TITLE
Copy-edit the Common Concepts section, except for the Recognition sub-section.

### LINKED ISSUE #317: Copy edit section A, Common Concepts

### PR DESCRIPTION
**Pull request by jyasskin**
Fixes #317, because I think we've already done a pass over Recognition.

***
<a href="https://pr-preview.s3.amazonaws.com/jyasskin/privacy-principles/pull/369.html#boring-terminology" title="Last updated on Dec 5, 2023, 8:05 PM UTC (950bcf8)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/369/c2806a2...jyasskin:950bcf8.html" title="Last updated on Dec 5, 2023, 8:05 PM UTC (950bcf8)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2022,73 +2022,62 @@
...
-interacts with a UI element on a web page, the <dfn>first party</dfn> of that interaction
-is usually the web page's [=page/first party=]. However, if a different [=actor=] controls
-how data collected with
-the UI element is used, and a reasonable person with a realistic cognitive budget would realize
+interacts with a part of a web page, the <dfn>first party</dfn> of that interaction
+is usually the web page's [=page/first party=].
+However, if a different [=actor=] makes the decisions about how that part of the
+page works, and a reasonable person with a realistic cognitive budget would realize
 that this other [=actor=] has this control, this other
 [=actor=] is the [=first party=] for the interaction instead.
```
Comment (jyasskin) [inline index.html:2059]: @dmarti, does this capture your concerns from #152?

File: index.html [hunk 2]
```diff
@@ -2098,21 +2087,23 @@
 * their location data,
 * an online identifier such as email or IP addresses,
 * browser fingerprints (based on a combination of
-configuration characteristics), or
-* factors specific to their physical, physiological, genetic, mental,
-economic,
-cultural, social, or behavioral [=identity=],
+  configuration characteristics), or
+* factors specific to their physical, physiological, genetic, mental, economic,
+  cultural, social, or behavioral [=identity=],
```
Comment (jyasskin) [inline index.html:2092]: I'm not sure these are all actually identifiers. They're ways to identify people, but we've defined an identifier as a thing that got assigned to a person.

File: index.html [hunk 3]
```diff
@@ -2122,41 +2113,39 @@
 destruction.
```
Comment (jyasskin) [inline index.html:2113]: Yikes! What parts of this do we actually need to list, and what parts could we simplify?

File: index.html (hunk 1 above)
Comment (dmarti) [reply index.html:2059]: @jyasskin Yes, that works for me, thank you.

File: index.html [hunk 4]
```diff
@@ -2022,73 +2022,62 @@
 [=people=] to refer to human beings, as a reminder of their humanity. When we use the term [=user=],
 it is to talk about the specific [=person=] who happens to be using a given system at that time.
 
-A <dfn data-lt="vulnerable">vulnerable person</dfn> is a [=person=] who may be unable to
-exercise sufficient self-determination in a [=context=]. Amongst other things, they should
+A <dfn data-lt="vulnerable">vulnerable person</dfn> in a particular [=context=]
+is a [=person=] whose ability to make their own choices can be taken away more
+easily than usual. Among other things, they should
```
Comment (jyasskin) [inline index.html:2027]: Filed #373 to keep thinking about whether we need this at all.

File: index.html [hunk 5]
```diff
@@ -2022,73 +2022,62 @@
...
+about the content and [=data processing=] on this [=origin=] or [=site=] is
 known as the web page's <dfn data-dfn-for="page">first party</dfn>. When a [=person=]
-interacts with a UI element on a web page, the <dfn>first party</dfn> of that interaction
-is usually the web page's [=page/first party=]. However, if a different [=actor=] controls
-how data collected with
-the UI element is used, and a reasonable person with a realistic cognitive budget would realize
+interacts with a part of a web page, the <dfn>first party</dfn> of that interaction
+is usually the web page's [=page/first party=].
+However, if a different [=actor=] makes the decisions about how that part of the
+page works, and a reasonable person with a realistic cognitive budget would realize
```
Comment (rhiaro) [inline index.html:2057]: is there a simpler / more self-explanatory phrase than "cognitive budget"?

File: index.html (hunk 2 above)
Comment (torgo) [reply index.html:2092]: As discussed today maybe these are characteristics rather than unique identifiers...

Comment (jyasskin) [reply index.html:2092]: Punting further discussion to #374.

File: index.html [hunk 6]
```diff
@@ -2122,41 +2113,39 @@
...
-perform the relevant [=processing=] if it:
+A <dfn data-lt="data processor">service provider</dfn> or [=data processor=]:
 
-* is processing the data on behalf of that [=actor=];
+* [=processes=] data on behalf of a [=data controller=];
 * ensures that the data is only retained, accessed, and used as directed by that
-  [=actor=] and solely for the list of explicitly-specified [=purposes=]
-  detailed by the directing [=actor=] or [=data controller=];
+  [=data controller=] and solely for the list of explicitly-specified [=purposes=]
+  detailed by the directing [=data controller=];
```
Comment (torgo) [inline index.html:2142]: @darobin can you review this change? It looked good to us on today's call but wanted your eyes on it as well.

File: index.html (hunk 3 above)
Comment (jyasskin) [reply index.html:2113]: Left for discussion in #375.

File: index.html (hunk 5 above)
Comment (jyasskin) [reply index.html:2057]: Hmmm, maybe:

```suggestion
page works, and a reasonable person with a realistic amount of time and energy to think about the situation would realize
```

Although I'm a little worried that the sentence is getting long enough that "would realize" is disconnected from its subject.

Comment (wseltzer) [reply index.html:2057]: I thought about flipping the order of the clauses, but that doesn't make it more readable either. You could omit "to think about the situation." 
```suggestion
page works, and a reasonable person with a realistic amount of time and energy would realize
```

File: index.html [hunk 7]
```diff
@@ -2122,41 +2113,39 @@
...
 A <dfn>data controller</dfn> is an [=actor=] that determines the [=means=] and [=purposes=]
 of data processing. Any [=actor=] that is not a [=service provider=] is a [=data controller=].
 
-A <dfn data-lt="data processor">service provider</dfn> or [=data processor=] is considered to be in
-the same category of [=first party=] or [=third party=] as the [=actor=] contracting it to
-perform the relevant [=processing=] if it:
+A <dfn data-lt="data processor">service provider</dfn> or [=data processor=]:
 
-* is processing the data on behalf of that [=actor=];
+* [=processes=] data on behalf of a [=data controller=];
```
Comment (darobin) [inline index.html:2139]: This is incorrect: you can have a service provider of a service provider, in fact it's very common.

File: index.html (hunk 6 above)
Comment (darobin) [reply index.html:2142]: This is also incorrect as it rules out the possibility of being the processor of a processor.

File: index.html [hunk 8]
```diff
@@ -2122,41 +2113,39 @@
...
-  detailed by the directing [=actor=] or [=data controller=];
+  [=data controller=] and solely for the list of explicitly-specified [=purposes=]
+  detailed by the directing [=data controller=];
 * may determine implementation details of the data processing in question but
   does not determine the [=purpose=] for which the data is being [=processed=]
   nor the overarching [=means=] through which the [=purpose=] is carried out;
 * has no independent right to use the data other than in a [=de-identified=] form (e.g., for
   monitoring service integrity, load balancing, capacity planning, or billing); and,
-* has a contract in place with the [=actor=] which is consistent with the above limitations.
+* has a contract in place with the [=data controller=] which is consistent with the above limitations.
```
Comment (darobin) [inline index.html:2148]: Same issue.

**Review by darobin — CHANGES_REQUESTED** [review]
This is generally good, but replacing actor with controller in the definition of a processor introduces a bug.

File: index.html (hunk 5 above)
Comment (darobin) [reply index.html:2057]: +1 to @wseltzer's version

File: index.html [hunk 9]
```diff
@@ -2022,73 +2022,62 @@
...
 with.
 
-The <dfn>Vegas Rule</dfn> is a simple implementation of privacy in which "<i>what happens with the
-[=first party=] stays with the [=first party=]</i>." Put differently, the [=Vegas Rule=] is followed
-when the [=first party=] is the only [=data controller=]. While the [=Vegas Rule=] is a good
-guideline, it's neither necessary nor sufficient for [=appropriate=] [=data processing=]. A [=first
-party=] that maintains exclusive access to a person's data can still [=process=] it
-[=inappropriately=], and there are cases where a third party can learn information about a person
-but still treat it [=appropriately=].
-
```
Comment (darobin) [inline index.html:2081]: Can we not revisit decisions we've discussed and made? See:
* https://github.com/w3ctag/privacy-principles/pull/77
* https://github.com/w3ctag/privacy-principles/blob/48f13c9b0110f9422041f8e0929711b27e793469/meetings/2021-11-17-minutes.md

File: index.html [hunk 10]
```diff
@@ -2022,73 +2022,62 @@
...
 ## Acting on Data {#acting-on-data}
 
 We define <dfn data-lt="data">personal data</dfn> as any information that is directly or
 indirectly related to an identified or identifiable [=person=], such as by reference to an
-[=identifier=] ([[GDPR]], [[OECD-Guidelines]], [[Convention-108]]).
+[=identifier=]. (This matches the [[[GDPR]]], the [[[OECD-Guidelines]]], and the [[[Convention-108]]].)
```
Comment (darobin) [inline index.html:2073]: I worry about "this matches" as that is expressing a legal opinion. Referencing documents that define PD is helpful as it allows folks to explore the space further if they need to, but claiming that our definition matches the exact sense used by three distinct legal documents really worries me.

File: index.html (hunk 7 above)
Comment (jyasskin) [reply index.html]: Fixed.

File: index.html (hunk 6 above)
Comment (jyasskin) [reply index.html]: Fixed.

File: index.html (hunk 8 above)
Comment (jyasskin) [reply index.html]: Fixed.

File: index.html (hunk 9 above)
Comment (jyasskin) [reply index.html:2081]: We re-discussed this today and decided to keep the Vegas Rule out.

File: index.html (hunk 10 above)
Comment (jyasskin) [reply index.html]: I've un-done this change.

**Review by torgo — APPROVED** [review]
as agreed in today's TF call

**Pull request merged**